src/locale/locales/de/messages.po

```diff
@@ -951,15 +909,15 @@ msgstr "Ein neues Konto erstellen"
 
 #: src/view/screens/Settings/index.tsx:384
 msgid "Create a new Bluesky account"
-msgstr ""
+msgstr "Erstellen Sie ein neues Bluesky-Konto"
 
 #: src/view/com/auth/create/CreateAccount.tsx:129
 msgid "Create Account"
 msgstr "Konto erstellen"
 
 #: src/view/com/modals/AddAppPasswords.tsx:226
 msgid "Create App Password"
-msgstr ""
+msgstr "App-Passwort erstellen"
```

Comment:
You have been using two different words here, "Passwort" and "Kennwort". If you want to be strictly German, "Kennwort" would be the better option. But most "modern" people know what "Passwort" means as well.

Reply:
I think when I looked at what had been translated previously they had arrived at 'App-Passwort' in other instances so I decided to stick to that.